### PR TITLE
Moves from the 'colors' package to 'chalk'.

### PR DESCRIPTION
**Pull request by jbnicolai**
See discussion in: #275

Contains:
- b50244f Removes dependency on 'supports-color', now covered by 'chalk'.
- 9c7c8c2 Replaces the 'colors' package with 'chalk'.

### CODE REVIEW

**Comment by mrjoelkemp** [comment]
Lgtm

File: package.json [hunk 1]
```diff
@@ -66,8 +66,8 @@
     "node": ">= 0.10.0"
   },
   "dependencies": {
+    "chalk": "^1.0.0",
```
Comment (mikesherov) [inline package.json]: We prefer tilde, can you please change that?

Comment (jbnicolai) [inline package.json]: Fixed and `--amend`ed

**Comment by jbnicolai** [comment]
Updated.

**Comment by jbnicolai** [comment]
Anything I can do to get this merged? :)

**Comment by mikesherov** [comment]
@jbnicolai, except wait patiently? ;-)

Just haven't had time to do a merge / test run. Will get to it shortly.

**Comment by jbnicolai** [comment]
Heh, I'll stop nagging ;-)

**Comment by mikesherov** [comment]
@jbnicolai , thanks again!

**Comment by jbnicolai** [comment]
high five

**Pull request closed without merging**